modules/torch/__init__.py

```diff
@@ -94,23 +119,29 @@ def update_led(self):
         else:
             self.led[0] = (0, 0, 0)
         self.led.write()
-        self.update_screen(full=False)
+        # Only update the screen if we're in the foreground
+        if self.is_active:
+            self.update_screen(full=False)
 
     def toggle_led(self):
+        self.flash_stop()
```

Comment:
Similar to the brightness thing, it'd be really good if this could be used to toggle flashing on and off, ie if flashing, it turns off the LED, but remembers the flash mode, such that next time it'll resume flashing